appserver/payara-appserver-modules/microprofile/fault-tolerance/src/main/java/fish/payara/microprofile/faulttolerance/FaultToleranceServiceConfiguration.java

```diff
@@ -39,22 +39,38 @@
  */
 package fish.payara.microprofile.faulttolerance;
 
+import javax.validation.constraints.Min;
+
 import org.glassfish.api.admin.config.ConfigExtension;
 import org.jvnet.hk2.config.Attribute;
 import org.jvnet.hk2.config.Configured;
 
 /**
  * Configuration for the Fault Tolerance Service.
- * @author Andrew Pielage
+ *
+ * @author Andrew Pielage (initial)
+ * @author Jan Bernitt (change to pool size)
  */
 @Configured(name = "microprofile-fault-tolerance-configuration")
 public interface FaultToleranceServiceConfiguration extends ConfigExtension {
 
-    @Attribute(defaultValue = "concurrent/__defaultManagedExecutorService", dataType = String.class)
```

Comment:
OK just did a quick check and no it doesn't throw an error if the domain.xml has unrecognised entries on startup.

Reply:
I this good news, meaning I can just erase them?

Reply:
The only thing that's stopping me from saying _yes_ is "what happens when someone updates from 201 and they were using this to use a non-default executor".
It might be nice to check if it's been configured to use a non-default executor on startup and print a warning saying it's not used any more - cleanest way I can think of to do this would be to retain the attribute and read it using the get method rather than doing xml parsing

Reply:
I kept the fields and added the `obsolete = true`.